tt_metal/hw/ckernels/blackhole/metal/llk_api/llk_sfpu/ckernel_sfpu_reshuffle_rows.h

```diff
@@ -0,0 +1,23 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
```

Comment:
```suggestion
// SPDX-FileCopyrightText: © 2025 Tenstorrent AI ULC
```
This header should conform with the new license. Also, since this is a new file, year should be 2025.

Reply:
I guess so, but I would expect some find and replace PR where this would be changed once and for all, so no bother.